src/views/RoofSelectedInformation.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+import UiSearchBar from '@/components/ui/UiSearchBar.vue'
+import UiCharDonut from '@/components/ui/UiChartDonut.vue'
+</script>
+
+<template>
+  <UiSearchBar></UiSearchBar>
```

Comment:
Looking at the design, I think we need to put the `UiSearchBar` outside the side panel since there is a mock-up where the `UiSearchBar` is seen when the side panel is collapsed. Or there is a way to do that without removing the `UiSearchBar` from the side panel?

Reply:
I have already implement the page where the UiSearchBar is outside the navbar , in this PR : https://github.com/sigrennesmetropole/cooperation_jn_solar/pull/18
For this view i have place the UiSearchBar in the App.vue component
Do you think we have do delete the UiSearchBar on all components expect App.vue ?

Reply:
On second thought, we can use the same smart component for this component, and my component (https://github.com/sigrennesmetropole/cooperation_jn_solar/pull/20). Because they will need the same source/data (i.e. roof area)

Reply:
> Do you think we have do delete the UiSearchBar on all components expect App.vue ?

I guess so. In my opinion, the UiSearchBar should be only in one place.